### PR TITLE
Landing page content accessibility fixes

### PR DESCRIPTION
**Pull request by cehsu**
These changes resolve two landing page content accessibility issues:
- ['Skip to Content' link lacks focus](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/6629)
screen shot 2018-01-01 at 3 10 43 pm

- [Feedback Link is Out of Order](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/6633)
In this case the link was removed. This link seemed to serve the purpose of allowing users to skip to later content, however, it was not provided with a visual expression (unlike the skip to content link at the top), so rather than providing it with a visual expression, as this seem like it would require design work, I have removed it here for now. If we would like to include a visual expression and skip to feedback link here, I will create a ticket for this work.

### CODE REVIEW

**Comment by jbalboni** [comment]
@cehsu I'm not sure I understand how the skip to content link issue is fixed. It looks like we're adding padding and that's it?

File: src/sass/base/_va.scss [hunk 1]
```diff
@@ -21,6 +22,10 @@ body {
 }
 
 // Skip link
+.show-on-focus-wrapper {
+  padding: 1em;
+}
```
Comment (ncksllvn) [inline src/sass/base/_va.scss]: Looks like this padding is showing when at all times, instead of just when focused.

Comment (cehsu) [reply src/sass/base/_va.scss]: Good catch @ncksllvn, fixed.

**Comment by cehsu** [comment]
Nice catch, @ncksllvn, the content id has been added to the home layout in a manner consistent with that of the home-no-search layout.

**Comment by cehsu** [comment]
Thanks @jbalboni, I've added a margin instead in light of your question.

**Comment by jbalboni** [comment]
@cehsu Any idea why these links below the buttons aren't aligned anymore? It looks correct on staging/prod for me, but not on the review instance.

screen shot 2018-01-03 at 9 33 31 am

**Comment by ncksllvn** [comment]
The homepage has a `va-homepage-benefits usa-content` class around those links. If you delete the `usa-content` class I think it aligns correctly.

image

**Comment by cehsu** [comment]
Thanks for the fix @ncksllvn, kindly refer to this updated screenshot:
 screen shot 2018-01-03 at 11 33 21 am

**Review by ncksllvn — APPROVED** [review]
Perfect!

File: content/includes/footer.html [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <footer class="footer" role="contentinfo">
   <div class="footer-inner">
-    <a class="sr-only" href="#feedback-tool" name="feedback-tool">Give feedback on this page</a>
```
Comment (cehsu) [inline content/includes/footer.html]: @ncksllvn is this change consistent with your updates to the feedback tool and a reasonable way to resolve the link order issue that is referenced here?

**Review by jbalboni — APPROVED** [review]

File: content/includes/footer.html (hunk 2 above)
Comment (ncksllvn) [reply content/includes/footer.html]: Yeah that looks good! That's consistent with the feedback tool. The only thing I had to watch out for was that the FAQ page contains this in its content -

` <a href="#feedback-tool" onclick="document.getElementById('feedback-tool').click()">Provide feedback on Vets.gov</a>.`

But this work isn't affected by that.

Comment (cehsu) [reply content/includes/footer.html]: Thanks for the heads up @ncksllvn.

Comment (ncksllvn) [reply content/includes/footer.html]: Thank you for your great work!

**Pull request merged**